dali/base/dadfs.hpp

```diff
@@ -939,15 +937,16 @@ inline cost_type getReadCost(const IPropertyTree & fileAttr, Source source)
     if (fileAttr.hasProp(getDFUQResultFieldName(DFUQRFreadCost)))
         return fileAttr.getPropInt64(getDFUQResultFieldName(DFUQRFreadCost), 0);
     else
+        return calcReadCost(fileAttr, source);
+}
+
+template<typename Source>
+inline cost_type calcWriteCost(const IPropertyTree & fileAttr, Source source)
```

Comment:
similarly, to be explicit what this func is for, might be good to rename it to calcLegacyIndexWriteCost